ChromecastSampleApp/ChromecastSampleApp/Players/PlayerViewController.m

```diff
@@ -53,6 +53,18 @@ - (void)viewDidLoad {
   self.castManager = [OOCastManagerFetcher fetchCastManager];
   self.castManager.delegate = self;
   
+  // Add custom parameters
+  
+  NSMutableDictionary<NSString *, NSString *> *customParams = [NSMutableDictionary new];
+  
+  customParams[@"userName"] = @"User";
+  customParams[@"initialVolume"] = @"0";
```

Comment:
would this start the video on the receiver always muted? Would prefer if by default we set it to `100` instead of `0`.

Reply:
I agree. Set the ```initialVolume``` to ```1```